lib/common.sh

```diff
@@ -105,7 +104,6 @@ IPAM_BASE_URL="${IPAM_BASE_URL:-metal3-io/ip-address-manager}"
 IPAMREPO="${IPAMREPO:-https://github.com/${IPAM_BASE_URL}}"
 IPAMBRANCH="${IPAMBRANCH:-master}"
 
-CAPIPATH="${CAPIPATH:-${M3PATH}/cluster-api}"
 CAPI_BASE_URL="${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}"
 CAPIREPO="${CAPIREPO:-https://github.com/${CAPI_BASE_URL}}"
```

Comment:
It seems to me, we can remove `CAPIREPO` too, since we are not using it anywhere, WDYT?

Reply:
Done.